adm/src/blockstore.rs

```diff
@@ -15,12 +15,12 @@
  * ------------------------------------------------------------------------------
  */
 
-use proto::block::{Block, BlockHeader};
+use crate::proto::block::{Block, BlockHeader};
```

Comment:
While you're at it, can you move this next to the other `use crate::...` imports?

Reply:
Fixed